src/birdie/internal/titles.gleam

```diff
@@ -446,8 +456,14 @@ fn try_fold_fields(
   fun: fn(a, glance.Expression) -> Result(a, b),
 ) -> Result(a, b) {
   use acc, field <- list.try_fold(over: fields, from: acc)
-  let glance.Field(item: expression, label: _) = field
-  try_fold_expression(expression, acc, fun)
+  case field {
+    glance.LabelledField(item: expression, label: _) ->
+      try_fold_expression(expression, acc, fun)
+    glance.UnlabelledField(item: expression) ->
+      try_fold_expression(expression, acc, fun)
+    glance.ShorthandField(label: _) ->
+      panic as "No expression in shorthand field"
```

Comment:
This should not panic and instead stop the folding, otherwise birdie would panic every time someone uses a label shorthand in their code

Reply:
This now returns `Ok(acc)` instead ending the fold